### ISSUE TITLE
Can't extract  force and displace with alamode tools

### ISSUE THREAD

**Issue by yingxiangcai** [ttadano/alamode#1]
I have some problem to extract force and displace while I type "python extract.py --VASP=POSCAR --get=disp vasprun*.xml > disp.dat" in the terminal. And the error is "no module named displace". So I check the extract.py and find a sentence of "from displace import *". I try put the directory of tools into the PYTHONPATH, but It still doesn't work.  Could you help me to extract the force and displace correctly. Looking forward to your responce.

**Comment by ttadano** [comment]
Hi. I've found your procedure works perfectly in my environment.
I have no idea why it's now working in your case. Probably, you may have some typos in the `PYTHONPATH`.
If not, please consider adding the tools directory to the `PATH` (not `PYTHONPATH`) as follows:
```
export PATH=${ALAMODE_root}/tools:$PATH
```
Then, you can use the script simply as
```
$ extract.py --VASP POSCAR --get disp vasprun*.xml > disp.dat
```

**Comment by ttadano** [comment]
closed #1
